lib/generate_speaker_list/generator.rb

```diff
@@ -0,0 +1,52 @@
+require 'json'
+
+module GenerateSpeakerList
+  class Generator
+    SPEAKERS_DATA_FILE = 'data/speakers.json'
+
+    def initialize
+      @files = Dir['source/meetings/*/*/index.html.md'].sort.reverse
+      @talks = []
+      @authors = {}
+    end
+
+    def call
+      load_all_talks_from_files!
+
+      @talks = @talks.flatten.select { |talk| talk[:author] != nil }.group_by { |talk| talk[:author] }
```

Comment:
There's nothing in the file for talks from before 2010.  I know the structure of the write-ups has changed a bit, but not by a huge amount I don't think.  So it's weird there's nothing in there from those early meetings.

In the json file I've spotted a few weird speaker names where we've got a greedy regexp match on names with multiple links in them.  So I wonder if we should revisit the parsing approach in general?  Would we get better results if we rendered the page from markdown -> HTML and then parse with nokogiri?  Or maybe we render into Kramdowns internal structure and walk parse that?